src/main/groovy/com/lesfurets/jenkins/unit/BasePipelineTest.groovy

```diff
@@ -140,7 +164,18 @@ abstract class BasePipelineTest {
                 updateBuildStatus('UNSTABLE')
             }
         })
+        helper.registerAllowedMethod("withCredentials", [Map, Closure])
         helper.registerAllowedMethod("withCredentials", [List, Closure], withCredentialsInterceptor)
+        helper.registerAllowedMethod('withEnv', [List, Closure], { List list, Closure c ->
+            list.each {
+                //def env = helper.get
+                def item = it.split('=')
+                assert item.size() == 2, "withEnv list does not look right: ${list.toString()}"
```

Comment:
it's possible to have additional '=' inside the item, e.g. when passing GRADLE_OPTS. Does it make sense to have this validation in the common library?